src/nvim/api/vim.c

```diff
@@ -663,6 +663,115 @@ Array nvim_get_api_info(uint64_t channel_id)
   return rv;
 }
 
+/// Call many api functions atomically
+///
+/// This has two main usages: Firstly, to perform several requests from an
+/// async context atomically, i.e. without processing requests from other rpc
+/// clients or redrawing or allowing user interaction in between. Note that api
+/// methods that could fire autocommands or do event processing still might do
+/// so. For instance invoking the :sleep command might call timer callbacks.
+/// Secondly, it can be used to reduce rpc overhead (roundtrips) when doing
+/// many requests in sequence.
+///
+/// @param calls an array of calls, where each call is described by an array
+/// with at least three elements: the request name, an array of arguments, and
+/// a boolean indicating if the return value should be sent back.
+/// @param[out] err Details of a validation error of the nvim_multi_request call
+/// itself, i e malformatted `calls` parameter. Errors from called methods will
+/// be indicated in the return value, see below.
+///
+/// @return an array with three elements. The first is the number of calls that
+/// where performed without errors. If it is the same as the length of "calls",
+/// all calls were successful, otherwise it is the number of the last attempted
```

Comment:
I'm confused by this. It says the first item is `number of calls performed without errors`, then says if an error occurred the first item is the `call that resulted in an error`.